### PR TITLE
Update pinned certs Cocoapod

### PR DESCRIPTION
**Pull request by jayzes**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -44,72 +44,6 @@ Once you have staging and production client credentials, you can switch between
 - Provisioning Profile `iOSTeam Provisioning Profile: co.ello.ElloDev`
 
 
-### Universal Links
```
Comment (steam) [inline Readme.md]: Why was the universal link documentation removed?

Comment (jayzes) [inline Readme.md]: The only reason I moved that too is that it involves signing with the certs, which aren't in this project anymore. I could split this section though - leave the structure here and signing instructions there - if that's preferable?

Comment (steam) [inline Readme.md]: My only concern is "losing" it in the other repo. I'm not sure it'll be as easily accessible over there. My preference would be to leave the heart of the universal link instructions here and reference the signing instructions in the cert repo. :metal:

**Comment by jayzes** [comment]
@steam updated in https://github.com/ello/ello-ios/pull/104/commits/610e6a0b4ad160872fe6413fa3da5a654d94c905

**Comment by steam** [comment]
👍

**Pull request merged**